src/components/Atoms/Input/index.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useRef, useEffect } from "react";
+import styled from "styled-components";
+import COLOR from "../../../variables/color.js";
+import FONTFAMILY from "../../../variables/font_family.js";
+import TEXT from "../../../variables/texts";
+
+export const Input = ({ onEditComplete, defaultValue }) => {
+  const inputRef = useRef(null);
+
+  const onkeypress = (e) => {
+    if (e.key === "Enter") {
+      inputRef.current.onblur();
+      inputRef.current.onkeypress("blur", () => {
+        onEditComplete(inputRef.current.value);
+        onkeypress(inputRef.current.value);
+      });
```

Comment:
```suggestion
      inputRef.current.onkeypress(() => {
        onEditComplete(inputRef.current.value);
        onkeypress(inputRef.current.value);
      });
```
`on〇〇` というプロパティの第一引数には、イベントが発火したときに実行される関数を登録します。

```suggestion
      inputRef.current.addEventListener("blur", () => {
        onEditComplete(inputRef.current.value);
        onkeypress(inputRef.current.value);
      });
```
第一引数に `"blur"` を設定するのは、 addEventListener のほうの書き方です。
両者とも同じようにイベントハンドラを設定できるのですが、書き方が大きく違うので注意してください。